AdLerDokumentation/Writerside/topics/PausenempfehlungGE.md

```diff
@@ -0,0 +1,33 @@
+# Pausenempfehlung
+
+<link-summary rel="summary"/>
+<card-summary rel="summary"/>
+<web-summary rel="summary"/>
+
+
+<table>
+    <tr>
+        <td></td>
+        <td>Beschreibung</td>
+    </tr>
+    <tr>
+        <td>Akronym</td>
+        <td>PE</td>
+    </tr>
+    <tr>
+        <td>Begriff</td>
+        <td>Pausenempfehlung</td>
+    </tr>
+    <tr>
+        <td>Definition</td>
+        <td id="summary">Eine Pausenempfehlung gibt konkrete Handlungsempfehlungen für die Pausenzeit. Die Pausenempfehlung wird dem Nutzer in einem Modal dargestellt. </td>
+    </tr>
+   <tr>
+        <td>Synonyme</td>
+        <td></td>
+    </tr>
+   <tr>
+        <td>Verwandte Begriffe</td>
```

Comment:
Bitte in Der Glossar schablone keine neuen Beschreibungsmöglichkeit ohne Abklärung hizufügen, (aber es gibt ja eh die neue schablone